src/test/java/karate/sign_up/Sign_up.java

```diff
@@ -0,0 +1,10 @@
+package karate.sign_up;
+
+import com.intuit.karate.junit5.Karate;
+
+public class Sign_up {
+    @Karate.Test
+    Karate testSample() {
+        return Karate.run("classpath:karate/sign_up/sign_up.feature").relativeTo(getClass());
+    }
+}
```

Comment:
remember you don't need to specify the feature.. why don't you delete this classes? What are you using that for?